main.js

```diff
@@ -68,7 +74,8 @@ const getAppContainer = (options) => {
 	if (options.withAb) {
 		logger.warn({
 			event: 'WITHAB_OPTION_DEPRECATED',
-			message: 'The \'withAb\' option is deprecated and no longer supported by n-express or n-flags-client'
+			message:
+				"The 'withAb' option is deprecated and no longer supported by n-express or n-flags-client"
```

Comment:
This looks like prettier or something edited an unrelated bit of code. The config seems wrong because ESLint is complaining about double quotes.

Reply:
bike shedding territory:
- i think it's fine to include a prettier change in an unrelated commit, nbd imo
- eslint is wrong in this case, prettier [chooses double quotes here because it's more readable](https://prettier.io/docs/en/rationale#strings) and eslint shouldn't be complaining about that. should probably be using the [eslint prettier plugin](https://github.com/prettier/eslint-config-prettier) to avoid these conflicts.